packages/front-end/components/Settings/PaymentInfo.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState, useEffect, useCallback } from "react";
+import { Flex, Text } from "@radix-ui/themes";
+import { CiCreditCard1 } from "react-icons/ci";
+import { Card } from "shared/src/types/subscriptions";
+import { useAuth } from "@/services/auth";
+import { useUser } from "@/services/UserContext";
+import track from "@/services/track";
+import { GBAddCircle } from "../Icons";
+import Callout from "../Radix/Callout";
+import Badge from "../Radix/Badge";
+import MoreMenu from "../Dropdown/MoreMenu";
+import DeleteButton from "../DeleteButton/DeleteButton";
+import Modal from "../Modal";
+import LoadingSpinner from "../LoadingSpinner";
+import CreditCardModal from "./CreditCardModal";
+
+export default function PaymentInfo() {
+  const [cardModal, setCardModal] = useState(false);
+  const [defaultCard, setDefaultCard] = useState<string | undefined>(undefined);
+  const [error, setError] = useState<string | undefined>(undefined);
+  const [cardData, setCardData] = useState<Card[]>([]);
+  const [loadingCards, setLoadingCards] = useState(false);
+  const { subscription } = useUser();
+  const { apiCall } = useAuth();
+
+  const customerId = "cus_Rg3aee6F7wi9EH"; //MKTODO: Fix this so the customer is from the subscription
```

Comment:
This is here as I need to make a change to the parent branch `jd/orb-billing` in order to persist the `customerId`